lib/collection/src/operations/validation.rs

```diff
@@ -86,13 +86,27 @@ fn describe_error(
             Some(value) => format!("value {value} invalid, must not be empty"),
             None => err.to_string(),
         },
+        "no_bracket_syntax" => match params.get("value") {
```

Comment:
Why do you need to add this case?

I thought it was removed in https://github.com/qdrant/qdrant/pull/2223

Reply:
Likely an accidental incorrect merge, because I don't think it is used.

Reply:
removed.